lib/ts/recipe/session/types.ts

```diff
@@ -204,17 +200,27 @@ export type RecipeInterface = {
     }): Promise<SessionClaimValidator[]> | SessionClaimValidator[];
 
     getSession(input: {
-        req: BaseRequest;
-        res: BaseResponse;
-        options?: VerifySessionOptions;
+        accessToken: string;
+        antiCsrfToken?: string;
+        options?: Omit<VerifySessionOptions, "sessionRequired">;
         userContext: any;
-    }): Promise<SessionContainerInterface | undefined>;
+    }): Promise<
+        | { status: "OK"; session: SessionContainerInterface }
+        | { status: "TOKEN_VALIDATION_ERROR"; error: any }
```

Comment:
```ts

async function createNewSessionWithoutRequestResponse(req: express.Request, resp: express.Response) {
    const userId = "user-id"; // This would be fetched from somewhere

    const result = await Session.createNewSessionWithoutRequestResponse(userId);

    const tokens = result.session.getAllSessionTokensDangerously();
    if (tokens.accessAndFrontTokenUpdated) {
        resp.set("st-access-token", tokens.accessToken);
        resp.set("front-token", tokens.frontToken);
        resp.set("st-refresh-token", tokens.refreshToken);
        if (tokens.antiCsrfToken) {
            resp.set("anti-csrf", tokens.antiCsrfToken);
        }
    }
}

async function refreshSessionWithoutRequestResponse(req: express.Request, resp: express.Response) {
    const refreshToken = req.headers.authorization?.replace(/^Bearer /, "");

    if (!refreshToken) {
        // This means that the user doesn't have an active session
        return resp.status(401);
    } else {
        const result = await Session.refreshSessionWithoutRequestResponse(refreshToken, true);
        if (result.status !== "OK") {
            return resp
                .status(401)
                .set("st-access-token", "")
                .set("set-refresh-token", "")
                .set("front-token", "remove"); // Or equivalent...
        }
        const tokens = result.session.getAllSessionTokensDangerously();
        if (tokens.accessAndFrontTokenUpdated) {
            resp.set("st-access-token", tokens.accessToken);
            resp.set("front-token", tokens.frontToken);
            resp.set("st-refresh-token", tokens.refreshToken);
            if (tokens.antiCsrfToken) {
                resp.set("anti-csrf", tokens.antiCsrfToken);
            }
        }
    }
}
```